Products/ZenRRD/zencommand.py

```diff
@@ -502,30 +526,24 @@ def doTask(self):
                         self._devId, "timeout",
                     )
 
-        except Exception as e:
-            self.state = TaskStates.STATE_PAUSED
-            err_msg = (
-                "Task paused  device=%s ip=%s interval=%s message=%s" % (
-                    self._devId,
-                    self._manageIp,
-                    self.interval,
-                    e.message,
-                )
-            )
-            if log.isEnabledFor(logging.DEBUG):
-                log.exception(err_msg)
-            else:
-                log.error(err_msg)
-            self._eventService.sendEvent(
-                STATUS_EVENT,
-                device=self._devId,
-                summary=e.message,
-                component=COLLECTOR_NAME,
-                severity=Error,
+        except defer.CancelledError:
```

Comment:
I'm not convinced that just catching CancelledError is a complete fix.  We don't know why the deferred was cancelled.  Simply logging "deferred was cancelled" provides no useful information to whoever is troubleshooting the message.